types/gifencoder/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6", "DOM"],
```

Comment:
Do you need a DOM reference to make it working? (for a `.d.ts` definition file that is)

Reply:
Yes, the main method to add frames to the gif takes Canvas contexts. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47303/files#diff-82143f2d9bf49fd0be45ef359156961cR22